app/main.py

```diff
@@ -1 +1,44 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int, is_hungry: bool = True):
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self):
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
remove redundant else statement

Reply:
Tests require to return 0 if animal isn't hungry

Reply:
Yes, but u don't need esle to do so

Reply:
Yes, exactly. Now I understand what you meant